packages/simulator/lib/dispatch/taxiDispatch.js

```diff
@@ -0,0 +1,60 @@
+const { toArray, map, filter, mergeMap, take } = require('rxjs/operators')
+const { plan } = require('../vroom')
+const moment = require('moment')
+
+const bookingToShipment = ({ id, pickup, destination }, i) => ({
+  id: i,
+  description: id,
+  amount: [1],
+  pickup: {
+    time_windows: pickup.timeWindow?.length
+      ? [
+          [
+            moment(pickup.timeWindow[0]).unix(),
+            moment(pickup.timeWindow[1]).unix() + 1,
+          ],
+        ]
+      : undefined,
+    id: i,
+    location: [pickup.position.lon, pickup.position.lat],
+  },
+  delivery: {
+    id: i,
+    location: [destination.position.lon, destination.position.lat],
+    time_windows: destination.timeWindow?.length
+      ? [
+          [
+            moment(destination.timeWindow[0]).unix(),
+            moment(destination.timeWindow[1]).unix() + 1,
```

Comment:
- Why convert timewindows here with moment?
- Why add +1?

Reply:
Resolved IRL:
- moment to convert to unix time as vroom needs.
- +1 might be to avoid vroom crashes if timeWindow[0] === timeWindow[1]